.github/workflows/lint-test.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: helm/chart-testing-action@v1.0.0-rc.1
         with:
           command: lint
-          config: .github/workflows/ct.yaml
+          config: ct.yaml
```

Comment:
We could just remove the config line altogether, since this is one of defaults… but this is nice because it shows people what's going on as part of the demo (should they wish to move their own config file to another location) 👍

Reply:
Yeah, that what we previously had. I think it makes sense to leave it in.